src/components/Banner/redirect-to-dbot.js

```diff
@@ -0,0 +1,12 @@
+import GTM from '@utilities/integrations/gtm';
+import { getActiveLoginId } from '@storage';
+
+export const visitDerivBot = () => {
+    const user_id = getActiveLoginId();
+    GTM.pushDataLayer({ event: 'bbot_moved_to_deriv_bot', user_id: user_id ?? null });
+    const isStaging = process.env.NODE_ENV === 'staging';
+    if (isStaging) {
+        window.open('https://staging-app.deriv.com/bot/?redirect_from_bbot=1', '_self', 'noopener');
+    }
+    window.open('https://app.deriv.com/bot/?redirect_from_bbot=1', '_self', 'noopener');
```

Comment:
Can we use `getRelatedDeriveOrigin()` to get the origin and then add `redirect_from_bbot=1` ?
```suggestion
  const { origin } = getRelatedDeriveOrigin();
  const url = origin + '?redirect_from_bbot=1';
  window.open(url);
```

Reply:
Sure. @sandeep-deriv